docs/layers.md

```diff
@@ -1048,7 +1048,7 @@ To improve performance, some road segments are merged at low and mid-zooms. To f
 * `min_zoom`: a suggestion for which zoom to draw a feature. The value is a float.
 * `ref`: Commonly-used reference for roads, for example "I 90" for Interstate 90. To use with shields, see `network` and `shield_text`. Related, see `symbol` for pistes.
 * `all_networks` and `all_shield_texts`: All the networks of which this road is a part, and all of the shield texts. See `network` and `shield_text` below. **Note** that these properties will not be present on MVT format tiles, as we cannot currently encode lists as values.
-* `network`: eg: `US:I` for the United States Interstate network, useful for shields and road selections. This only contains _road_ network types. Please see `bicycle_network` and `walking_network` for bicycle and walking networks, respectively.
+* `network`: eg: `US:I` for the United States Interstate network, useful for shields and road selections. This only contains _road_ network types. Please see `bicycle_network` and `walking_network` for bicycle and walking networks, respectively. Note that networks may include "modifier" information, for example `US:I:Business` for a business route or `US:I:Truck` for a truck route.
```

Comment:
Since we're whitelisting those in code instead of pass-thru, let's list the values in alphabetical order here (reorder from the SQL below):

`('Business', 'Spur', 'Truck', 'Alternate', 'Bypass', 'Connector', 'Historic', 'Toll', 'Scenic')`

Reply:
(Noting here there isn't a good OSM Wiki page for modifier to link to, so quoting it alone suffices.)